### PR TITLE
set up backend coffee filter

### PR DESCRIPTION
**Pull request by uwgnol1612**

### CODE REVIEW

File: server/schema/types/root_query_type.js [hunk 1]
```diff
@@ -22,12 +22,14 @@ const CoffeeShop = mongoose.model("coffeeShops")
 const selectorInput = new  GraphQLInputObjectType({
   name: 'Selectors',
   fields: {
-    type: { type: GraphQLString },
```
Comment (hegelocampus) [inline server/schema/types/root_query_type.js:25]: Why did you remove the type selector? Did we take that off the attributes for coffee
shop?

Comment (hegelocampus) [reply server/schema/types/root_query_type.js:25]: Although I'm not sure if this is even being used any more so this may be a non-issue

**Review by hegelocampus — CHANGES_REQUESTED** [review]
I'm not sure about the status of the selectors but if they are no longer being used at all then my comments are a non-issue but you should make sure this won't cause conflicts with John's earlier PR before proceeding.

File: server/schema/types/root_query_type.js (hunk 1 above)
Comment (uwgnol1612) [reply server/schema/types/root_query_type.js:25]: I changed it back (update in the new branch -frontend favorite, but as John set up the search feature, we actually don't use  that in the backend anymore.

Comment (uwgnol1612) [reply server/schema/types/root_query_type.js:25]: Please just ignore this pull request as they will be updated in the new branch 'frontend-favorite'
